test/splitter.js

```diff
@@ -0,0 +1,160 @@
+const truffleAssert = require('truffle-assertions');
+const Splitter = artifacts.require("./Splitter.sol");
+
+contract('Splitter', async accounts => {
+
+    let gasCost = async receipt => {
```

Comment:
Since this is a function, it would be better to name it as such. `getGasCost`?
Also, I don't think you plan on changing it, so use `const` instead of `let`.